internal/controlplane/handlers_evalstatus.go

```diff
@@ -309,6 +309,7 @@ func buildRuleEvaluationStatusFromDBEvaluation(
 		RemediationStatus:      string(eval.RemStatus.RemediationStatusTypes),
 		RemediationLastUpdated: timestamppb.New(eval.RemLastUpdated.Time),
 		RemediationDetails:     eval.RemDetails.String,
+		RuleDisplayName:        eval.RuleTypeDisplayName,
```

Comment:
If no display name is set, let's fall back to the rultype name

Reply:
+1 -- this isn't a required field.

Reply:
Ah so drop the display name field altogether?